lorry/binding/base.go

```diff
@@ -495,3 +508,33 @@ func (ops *BaseOperations) LeaveMemberOps(ctx context.Context, req *bindings.Inv
 	opsRes["message"] = "left of the current member is complete"
 	return opsRes, nil
 }
+
+func (ops *BaseOperations) StartOps(ctx context.Context, req *bindings.InvokeRequest, resp *bindings.InvokeResponse) (OpsResult, error) {
+	opsRes := OpsResult{}
+	err := hypervisor.StartDBService()
+	if err != nil {
+		opsRes["event"] = OperationFailed
+		opsRes["message"] = fmt.Sprintf("start db service failed: %v", err)
+		return opsRes, err
+	}
+
+	opsRes["event"] = OperationSuccess
+	opsRes["message"] = "start db service success"
+	return opsRes, nil
+}
+
+func (ops *BaseOperations) StopOps(ctx context.Context, req *bindings.InvokeRequest, resp *bindings.InvokeResponse) (OpsResult, error) {
```

Comment:
Stop may timeout, so what's the policy for timeout process?

Reply:
yes, if stopDBService is called and db service is still running, StopOps will return an error to indicate waiting for db to stop, and the caller that receive this error should to retry the operation later.